internal/autodiscovery/discovery.go

```diff
@@ -31,7 +31,7 @@ func DiscoveryWrapper(discovery dsModels.ProtocolDiscovery, lc logger.LoggingCli
 	locker.busy = true
 	locker.mux.Unlock()
 
-	lc.Info(fmt.Sprintf("device discovery triggered"))
+	lc.Info(fmt.Sprintf("protocol discovery triggered"))
```

Comment:
How often could this message be logged? 
If very infrequent then `Info` is ok, but if it could be frequent then should be `Debug`

Reply:
it depends on user's configuration, but in this case I think `Debug` is more suitable